pyrk/th_system.py

```diff
@@ -1,46 +1,21 @@
+import math
+from th_component import THSuperComponent
 from utilities.ur import units
 
 
 class THSystem(object):
-    """This class handles calculations and data related to the
-    thermal hydraulics sub block
+
+    """This class models
+    - 1-D heat diffusion in spherical geometry, for heat
+    generation at any radius in the sphere,
+    - advective heat transfer by fluid
+    - convective heat transfer at the solid surface to fluid.
     """
 
     def __init__(self, kappa, components):
         self.kappa = kappa
         self.components = components
-
-    def dtempdt(self, component, power, omegas, t_idx):
-        to_ret = 0*units.kelvin/units.second
-        cap = (component.rho(t_idx)*component.cp*component.vol)
-        if component.heatgen:
-            to_ret += self.heatgen(component, power, omegas)/cap
-        for interface, area in component.cond.iteritems():
-            env = self.comp_from_name(interface)
-            to_ret -= self.conduction(t_b=component.T[t_idx],
-                                      t_env=env.T[t_idx],
-                                      k=component.k,
-                                      L=component.vol/area,
-                                      A=area)/cap
-        for interface, d in component.conv.iteritems():
-            env = self.comp_from_name(interface)
-            to_ret -= self.convection(t_b=component.T[t_idx],
-                                      t_env=env.T[t_idx],
-                                      h=d['h'],
-                                      A=d['area'])/cap
-        for interface, d in component.mass.iteritems():
-            env = self.comp_from_name(interface)
-            to_ret -= self.mass_trans(t_b=component.T[t_idx],
-                                      t_inlet=env.T[t_idx],
-                                      H=d['H'],
-                                      u=d['u'])
-        for interface, d in component.cust.iteritems():
-            env = self.comp_from_name(interface)
-            to_ret -= self.custom(t_b=component.T[t_idx],
-                                  t_env=env.T[t_idx],
-                                  res=d['res'])/cap
-
-        return to_ret.to('kelvin/second')
+        #THSystem.__init__(self, kappa, components)
```

Comment:
why is this commented? If it should be deleted, then delete the line. If it should be retained, then keep it. Don't leave it commented out.